### ISSUE TITLE
same dir, but different result by using "du -sh ."

### ISSUE THREAD

**Issue by ahnselina** [juicedata/juicefs#1014]
**What happened**:
the dir "ceph-15.2.15" in my mac file system, use command "du -sh .", show result:
```
root@ceph-15.2.15 % du -sh .
166M	.
```
count the files num:
```
root@ceph-15.2.15 % ls -lR|grep "^-" |wc -l
    8495
``` 
then, copy the dir "ceph-15.2.15" to juicefs, the result:
```
root@/tmp/jfs/ceph-15.2.15 % du -sh .
1.4G	.
```
**What you expected to happen**:
in fact, I expected the result is same，it should be 166MB.
**How to reproduce it (as minimally and precisely as possible)**:
just copy the dir to juicefs

**Environment**:
- JuiceFS version (use `./juicefs --version`) : juicefs --version
juicefs version 0.17.1 (2021-10-28T05:30:05Z 4d11c4b)
- OS (e.g: `cat /etc/os-release`): macOS 11.3.1 (20E241) 
- Kernel (e.g. `uname -a`): Darwin GIH-L-5089.local 20.4.0 Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:41 PDT 2021; root:xnu-7195.101.2~1/RELEASE_ARM64_T8101 x86_64
- Object storage (cloud provider and region): ceph s3
- Redis info (version, cloud provider managed or self maintained): local redis(127.0.0.1:6379/1)

Is this just a display bug or in fact, it's ok, just I don't understand the  principle ?

**Comment by davies** [comment]
I copied a directory with 5K files into JuiceFS, and the result of `du` is close to the result of local disk:

```
$ cp -r . /jfs/path/
$ du . /jfs/path/ -sh
609M	.
598M	/jfs/path/
$ find . | wc -l
4902
```

Is it possible that you are using JuiceFS Cloud? which is not compatible with `du`.

If not, could go into the sub-directories to find out which one contribute most of the difference?

**Comment by ahnselina** [comment]
> I copied a directory with 5K files into JuiceFS, and the result of `du` is close to the result of local disk:
> 
> ```
> $ cp -r . /jfs/path/
> $ du . /jfs/path/ -sh
> 609M	.
> 598M	/jfs/path/
> $ find . | wc -l
> 4902
> ```
> 
> Is it possible that you are using JuiceFS Cloud? which is not compatible with `du`.
> 
> If not, could go into the sub-directories to find out which one contribute most of the difference?

JuiceFS Cloud?  I use the local redis(127.0.0.1:6379) to store meta data, the remote ceph s3 to store data. Is this situation not compatible with ```du```?

**Comment by davies** [comment]
@ahnselina We found the root cause, will be fixed by #1016

**Comment by ahnselina** [comment]
> @ahnselina We found the root cause, will be fixed by #1016

OK, I see it, thanks
